Amended copy (#125)
alphagov/paas-product-pages: pages/index.mdx

## Deploy applications without infrastructure specialists

> You can get something up on the internet in minutes, without depending on a web operations team.

Reviewer: _get something up_ seems needlessly colloquial

> You can get something up on the internet in minutes, without depending on a web operations team.

Reviewer: ```suggestion
You can deploy your app to the internet in minutes, without depending on a web operations team.
```

Reviewer 2: Webops / Web Operations team is a term that should go away

Author: without depending on a team of site reliability engineers

?

Reviewer 2: yep, sounds good

<GridColumn width='two-thirds'>

## Save your team time and money

By using GOV.UK PaaS you may save money by:

-   avoiding the upfront costs of being forced to build or procure your own infrastructure or platform
-   avoiding daily maintenance costs from needing to employ infrastructure engineers to keep services securely patched, up to date and running efficiently

> -   needing less information assurance because the platform is consistent and assured by Cabinet Office

Reviewer 2: I'm not sure what `platform is consistent` is trying to say here.

Author: "needing less information assurance because the platform is accredited and assured by Cabinet Office"

?

Reviewer 2: sounds good

## Resilient and secure

GOV.UK PaaS is designed to meet the needs of government teams working on public-facing services by:

> -   being hosted domestically within the UK

Reviewer 2: Dublin isn't in the UK

Reviewer: Perhaps we should say hosted on the "British isles"? Or cut to the chase and say Dublin.

Author: UK _or_ Dublin ?

Author: being hosted domestically within the UK by default or Dublin by request? 

We don't start new orgs in Dublin

Reviewer 2: yep I think the latter works nicely
